config.json

```diff
@@ -1,7 +1,8 @@
 {
-	"scheme": "http",
-	"host": "localhost",
+	"protocol": "http",
```

Comment:
scheme and host are the w3c names used in https://www.w3.org/Addressing/URL/url-spec.txt. These should be named using standard terms in the config.json file to avoid confusion. scheme is the name of the protocol to use, host is the name of the node to bind to.

Reply:
I have made this change so that the JSON keys match the attributes in the https://nodejs.org/docs/latest-v10.x/api/url.html